### ISSUE TITLE
Option to display absolute coordinates in sources coming from genomic regions

### ISSUE THREAD

**Issue by dgruano** [manulera/ShareYourCloning_frontend#224]
I find it useful to see the absolute genomic coordinates when working with large chucks of DNA and even with genes, for instance, when working in parallel with other tools such as genome browsers or BLAST.

**Comment by manulera** [comment]
What kind of coordinates do you mean / where would you like to display  them ?

As it is the coordinates within the reference sequence are displayed on the source once it's completed.

You mean to say that the editor should display the absolute coordinates instead of going from 1 to len(seq)? It may be possible to do this with OVE, but even then the numbers risk to be too big to be displayed properly, don't you think?

<img width="421" alt="Screenshot 2024-08-06 at 10 21 52" src="https://github.com/user-attachments/assets/76972722-6fde-4cd2-914a-5240a6c18531">

**Comment by dgruano** [comment]
Yes! It's great that they are displayed next to the name of the chromosome sequence. What I meant is that it would be useful to have them also in OVE, as this may help sequence viewing and design. I have not played with OVE so I don't know whether  it would fit. I'll check!

On the other hand, I agree that in the source view it may be a bit tight (although we could use abbreviations such as "1.45 Mb"). This is also true for OVE.
